exercises/sub-list/SublistExample.swift

```diff
@@ -1,71 +1,72 @@
 import Foundation
 
-struct SubList {
+enum SublistKind: String {
```

Comment:
The string value of the enum isn't used anywhere. Can be changed to just `enum SublistKind {`.